modules/exploits/windows/local/cve_2020_1571_windows_setup_lpe.rb

```diff
@@ -0,0 +1,197 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+  include Msf::Post::Windows::Priv # Needed for is_system?
+  include Msf::Exploit::EXE # Needed for generate_payload_exe
+  include Msf::Post::Windows::FileSystem # Needed for create_symlink
+  include Msf::Exploit::FileDropper # Needed for register_file_for_cleanup
+  include Msf::Post::File # Needed for write_file
+  include Msf::Post::Windows::ReflectiveDLLInjection # Needed for inject_dll_into_process
+
+  def initialize(info = {})
+    super(
+        update_info(
+          info,
+          'Name' => 'Windows Setup Arbitrary File Deletion Local Privilege Escalation',
+          'Description' => %q{
+            This module exploits an oversight when patching CVE-2020-1571 where Microsoft correctly changed the Windows Setup program's behavior
+            so that it removed a Time-Of-Check-Time-Of-Use bug when creating the directory C:\$WINDOWS.~BT, however they forgot to check whether
+            or not this folder was a reparse point prior to deleting the directory. Attackers can exploit this vulnerability to delete arbitrary
+            folders on the system as the SYSTEM user, which can be used to gain code execution as SYSTEM via known DLL hijacking issues within
+            Windows.
+
+            Note that this module will only work on Windows 10 v1909 and prior. This is noted by the author who found this bug, and I'm guessing
+            this is due to the need for their to be a new version of Windows that one can upgrade to, and since v2004 is the latest at the time of
+            writing, there is no way to trigger the functionality needed to exploit this bug on that version of the operating system. Its possible
```

Comment:
Can you please rephrase this to be in the third person context and remove any hints of speculation.

Reply:
Removed the speculation and turned it into a more factual statement addressing the fact that the module may not cover the same targets as the Microsoft advisory states and just left it at that. Let me know if that needs further editing.